src/ai/backend/client/cli/vfolder.py

```diff
@@ -176,7 +176,13 @@ def info(name):
               help='Transfer the file with the given chunk size with binary suffixes (e.g., "16m"). '
                    'Set this between 8 to 64 megabytes for high-speed disks (e.g., SSD RAID) '
                    'and networks (e.g., 40 GbE) for the maximum throughput.')
-def upload(name, filenames, base_dir, chunk_size):
+@click.option('--override-storage-proxy',
+              type=StorageProxyAddressParamCheckType(), default=get_config().address_map,
```

Comment:
Calling `get_config()` here may have side-effects because it is executed at the time of importing this module. It is better to use a static, immutable constant here.
You may define a class variable to the configuration class (which is static) and refer it. In this case, we could utilize `APIConfig.DEFAULTS`.

Reply:
The reason I use `get_config()` is that environment variable should be used when the flag is not given.
Im afraid of any side-effect, so I'm gonna change it to immutable constant and check the environment variable in `cli` or `func` side.

Reply:
I changed the default value to `None`.
how about this?